lib/client-api/src/hooks.ts

```diff
@@ -251,3 +266,45 @@ export function useEffect(create: () => (() => void) | void, deps?: any[]): void
   const effect = useMemoLike('useEffect', () => ({ create }), deps);
   currentEffects.push(effect);
 }
+
+export interface Listener {
+  (...args: any[]): void;
+  ignorePeer?: boolean;
+}
+
+export interface EventMap {
+  [eventId: string]: Listener;
+}
+
+export function useChannel(eventMap: EventMap) {
+  const channel = addons.getChannel();
+  useEffect(() => {
+    Object.entries(eventMap).forEach(([type, listener]) => channel.on(type, listener));
+    return () => {
+      Object.entries(eventMap).forEach(([type, listener]) =>
+        channel.removeListener(type, listener)
+      );
+    };
+  }, Object.keys(eventMap));
+
+  return channel.emit;
+}
+
+export function useParameter<S>(parameterKey: string, defaultValue?: S): S | undefined {
+  /*
+    TODO: come up with a better way of reaching the storyStore without a global variable 
```

Comment:
Maybe introduce another module-scope mutable variable and set it inside `applyHooks`?

Reply:
I'll do that